eng/Versions.props

```diff
@@ -93,7 +93,12 @@
   </PropertyGroup>
   <PropertyGroup>
     <!-- Dependencies from https://github.com/NuGet/NuGet.Client -->
-    <NuGetVersioningPackageVersion>5.8.0</NuGetVersioningPackageVersion>
+    <NuGetBuildTasksPackageVersion>6.0.6-rc.4</NuGetBuildTasksPackageVersion>
+    <NuGetVersioningPackageVersion>$(NuGetBuildTasksPackageVersion)</NuGetVersioningPackageVersion>
```

Comment:
I'm guessing this is somehow set to the current one somewhere?

Reply:
Yeah, it's controlled by darc flow if the version.details.xml includes the component you want updated (which it does) and the property in the version.props file matches the right format of _Packagenamewithoutdots_+"PackageVersion". So basically, we already had darc flow in the xml file but there wasn't an entry in the versions.props that matches the right format to get updated.